recipes/somoclu/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "somoclu" %}
+{% set version = "1.7.4" %}
+{% set sha256 = "6182b688d9e58bd692d88f1d7edc9f4b536fba722417bec43b4b38eaebd13af9" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-python-{{ version }}.tar.gz
+  url:  https://github.com/peterwittek/somoclu/releases/download/1.7.4/somoclu-python-1.7.4.tar.gz
```

Comment:
If you want to stick to the github releases for whatever reason, replace the `1.7.4` bits with `{{ version }}` at least so they'll work when you update the version variable in the future. :)